www/blog/2023/06/19/karmashapes/index.md

```diff
@@ -0,0 +1,790 @@
+---
+layout: page
+title: Introducing Karmashapes
+published: false
+date: 2023-06-19
+permalink: /blog/2023/06/19/introducting-karmashapes/
+category: blog
+excerpt: "Thanks to the Karmashapes initiative, Who’s On First now provides the best open data for towns and villages in India."
+authors: [Justin Meyers, stepps00, nvkelso]
```

Comment:
Looking at recent blogposts, I assumed this was just bolded text? https://whosonfirst.org/blog/2023/06/07/state-of-the-gazetteer/

I could update this to [justinelliotmeyers](https://github.com/justinelliotmeyers) so these all match github usernames..

Reply:
This part of the blog seems to have some bit rot, but:

- https://whosonfirst.org/blog/authors/

Which, adding a missing trailing `/` because S3 is strict about this but normal web servers aren't:

- https://whosonfirst.org/blog/authors/nvkelso/

Which lists out all recent blog posts I've been involved in.

At Mapzen we'd need to update an author's metadata entry for Justin, too.

At the very least make his name a single world and all lowercase? This could be removing the space or replacing it with a hyphen... or sure use hit Github username?

Reply:
(There's some extra steps @thisisaaronland does to get this ready for merge to main, so partly we can hold until we know what to do...)

Reply:
I see a new author line for Justin, I think we're good here!